openedx/core/djangoapps/user_api/preferences/views.py

```diff
@@ -263,3 +270,50 @@ def delete(self, request, username, preference_key):
             return Response(status=status.HTTP_404_NOT_FOUND)
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ChangeToVerifiedMode(APIView):
+    """
+    Change the status to 'approved' in Software Secure Photo Verification model for the given user.
+    """
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
```

Comment:
I think for now only staff users should be allowed to perform this action, https://www.django-rest-framework.org/api-guide/permissions/#isadminuser

Reply:
@jfavellar90 de acuerdo, igualmente la petición se realiza al generar el access token con el secret key de ecommerce.